plugins/dummy_c/Makefile

```
@@ -11,17 +11,28 @@
 # specific language governing permissions and limitations under the License.
 #
 
-PLUGIN_INFO_H_DIR= ../../
-FALCOSECURITY_PLUGIN_H_DIR=../../sdk/cpp
+PLUGIN_INFO_H_DIR= ./
+FALCOSECURITY_PLUGIN_H_DIR=./sdk/
 CURL = curl
 
-all: nlohmann/json.hpp libdummy_c.so
+all: sdk nlohmann/json.hpp libdummy_c.so
 
 clean:
-	rm -rf nlohmann *.so
+	rm -rf nlohmann *.so sdk/
 
 nlohmann/json.hpp:
-	rm -rf nlohmann && mkdir nlohmann && $(CURL) -sLo nlohmann/json.hpp https://github.com/nlohmann/json/releases/download/v3.10.2/json.hpp
+	$(CURL) -sLo nlohmann/json.hpp https://github.com/nlohmann/json/releases/download/v3.10.2/json.hpp
```

```suggestion
	mkdir -p nlohmann && $(CURL) -sLo nlohmann/json.hpp https://github.com/nlohmann/json/releases/download/v3.10.2/json.hpp
```
The CI's test is currently failing for that :point_up: 

We need to ensure that the directory exists, otherwise, it will fail.